source/mongo.txt

```diff
@@ -0,0 +1,1249 @@
+.. _mongo:
```

Comment:
I'm pretty sure the mongo shell is its own package, and is not in the tools package.